client/webserver/site/src/js/ws.js

```diff
@@ -102,6 +103,7 @@ class MessageSocket {
 
       // Stub out standard functions
       conn.onclose = (evt) => {
+        window.log('ws', 'onclose')
```

Comment:
May wish to remove the `console.log` a few lines down, but I'd suggest logging evt.code with this since it'll be the websocket close code like 1000, 1001, 1006, etc.

Reply:
I'd like to leave the lower log, but maybe make it a `console.error` instead. Leaving this top one for the timestamp though.